### PR TITLE
add openjsf logo to footer and remove linux foundation references

### LINKED ISSUE #2805: Add OpenJS Foundation logo
As part of https://github.com/openjs-foundation/project-onboarding/issues/1, we need to add the OpenJS Foundation logo to our website. 

The artwork are available at https://github.com/openjs-foundation/artwork/tree/master/openjs_foundation

### PR DESCRIPTION
**Pull request by oliviaroche**
Fixed #2805 

Really rudimentary addition of the openjsf logo to the footer as per the open issue. I removed the linux foundation logo and text as well.

Screen Shot 2019-12-02 at 23 59 45

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
LGTM

**Comment by XhmikosR** [comment]
The images need optimization. Also, isn't the logo a little tall?

File: layouts/partials/footer.hbs [hunk 1]
```diff
@@ -3,10 +3,10 @@
 <footer {{#if className}} class="{{className}}"{{/if}}>
 
   <div class="container">
-    <div class="linuxfoundation-footer">
+    <div class="openjsfoundation-footer">
       <div class="issue-link-container">
-        <a class="linuxfoundation-logo" href="http://collabprojects.linuxfoundation.org">
-          <img alt="Linux Foundation Collaborative Projects" src="/static/images/lfcp.png" width="416" height="15">
+        <a class="openjsfoundation-logo" href="https://openjsf.org/">
+          <img alt="OpenJS Foundation" src="/static/images/openjs_foundation-logo.png" width="200">
```
Comment (XhmikosR) [inline layouts/partials/footer.hbs]: Please add the height too

Comment (XhmikosR) [reply layouts/partials/footer.hbs]: I was about to switch to the SVG logo here so we might as well do it in this branch. No need to even add a PNG IMHO

**Comment by XhmikosR** [comment]
Alright, I compressed the SVG and switched to it. I will proceed the other cleanups in another PR.

Live preview: <https://goofy-shockley-2da2af.netlify.com/en/>

The patches shouldn't be squashed.

/CC @Trott @MylesBorins for review. BTW we could *really* use PR previews :P

**Review by Trott — APPROVED** [review]

**Comment by XhmikosR** [comment]
Can someone check the build log on prod? It seems it's not live yet

**Comment by Trott** [comment]
> Can someone check the build log on prod? It seems it's not live yet

Check again, maybe? I see the OpenJS Foundation logo in the footer.

**Comment by Trott** [comment]
I do wonder if we should just remove the logo from the footer entirely. The OpenJS site itself has no logo in the footer and unless we can get a very long and flat logo, it's a weird place for a logo and is kind of visually distracting.

**Comment by XhmikosR** [comment]
Seems broken for me here...

I'm on mobile.

Maybe we missed something in the ngnix rules? Or it's a cloudflare issue

**Comment by XhmikosR** [comment]
We could also make it shorter if that works for you better.

**Comment by XhmikosR** [comment]
Ok now it's all good here too.

Feel free to ping me if you need any help with the logo. I personally
expressed my opinion that the logo does look a little tall.

**Pull request merged**